app/styles/components/az-bars.scss

```diff
@@ -0,0 +1,19 @@
+az-bars {
+  width: calc(100% - 96px);
+  div {
+    text-align: right;
+    padding: 5px;
+    height: 30px;
+    span {
+      width: 100%;
+      padding-right: 5px;
+      display: inline-block;
+      &.Creature {background-color: map-get($color-red, '300');}
```

Comment:
I was doing that for simple re-use of colors. I'm open to color suggestions. Also, what do you think of the az-bars tag?